### DISCUSSION TITLE
PullRequest 25991 - Is it a sub-map it is?

### DISCUSSION THREAD

**Discussion by JustZerooo** [TrinityCore/TrinityCore#25913] [General]
Why and why not is this pr shit? https://github.com/TrinityCore/TrinityCore/pull/25911 To me it is a logic think to have it there why you will not have it there?

**Comment by MaxtorCoder** [comment]
It's not logic, Argus is a Continent/Planet, not under Broken Isles.

**Reply by JustZerooo** [reply]
Okay and how you will know it and why? reading the whole file system means not it is. a planet eventually yes but why a seperate folder if the map system shows it as sub map? huh?

**Reply by jackpoz** [reply]
I always got confused when searching for the script of an instance/boss that I have to know the whole WoW lore to find the right folder structure.

**Reply by funjoker** [reply]
going by "own planet" stuff... There should be no Kalimdor, EasternKingdoms, etc in scripts folder.
Those would need to be moved to an own folder "Azeroth" then.

**Comment by funjoker** [comment]
Looking ingame on the Map they have this way of listing:

World:
- Azeroth
  - Kalimdor
    - All Zones
  - Eastern Kingdoms
    - All Zones
  - Northrend
    - All Zones
  - Maelstrom
    - Deepholm
    - Kezan
    - The Lost Isles
  - Pandaria
    - All Zones
  - Broken Isles
    - Dalaran
    - All Zones
    - **Argus** (top right corner, is listed under Broken Isles)
       ![image](https://user-images.githubusercontent.com/846960/105494952-cf361580-5cbb-11eb-9399-4fb25d80a986.png)
  - Kul Tiras
    - All Zones
    - NOTE: Nazjatar is listed on top right corner, but is own zone under Azeroth!
      ![image](https://user-images.githubusercontent.com/846960/105495042-f391f200-5cbb-11eb-94ae-ec35a6a35c9d.png)
  - Zandalar
    - All Zones
    - NOTE: Nazjatar is listed on top right corner, but is own zone under Azeroth!
      ![image](https://user-images.githubusercontent.com/846960/105495042-f391f200-5cbb-11eb-94ae-ec35a6a35c9d.png)
  - Nazjatar
    ![image](https://user-images.githubusercontent.com/846960/105495042-f391f200-5cbb-11eb-94ae-ec35a6a35c9d.png)
- Outland
    - All Zones
- Draenor
    - All Zones
- Shadowlands
    - All Zones


If you pick up blizzards way of presenting, Argus belongs under Broken Isles.
Nazjatar under Azeroth.

**Reply by JustZerooo** [reply]
That is what i would say and why i would move argus to that subfolder.

**Reply by offl** [reply]
Changing file name \ moving file also removes history of file

**Reply by funjoker** [reply]
You can go back to moe commit and from there back to the old path. jsut a bit more clicks

**Reply by jackpoz** [reply]
> also removes history of file

git log has "--follow" option to follow renames https://git-scm.com/docs/git-log#Documentation/git-log.txt---follow 

In general, the fear of renaming or moving a file because git log and git blame will require a bit more effort shouldn't stop us from moving files around if that's what we would like.

It's a feature that GitHub has not implemented but Git GUIs have (for example Tortoise Git). GitHub issue is tracked at https://github.com/isaacs/github/issues/900

**Reply by ghost** [reply]
Thanks for the interesting info. Looking forward to see how and when it will be implemented on this platform.

**Comment by Faq** [comment]
Didn't catch a result of this discussion. It a Yes or No for that PR as it kinda follows Blizz logic?

**Reply by MaxtorCoder** [reply]
I don't see a reason to fuck over git history.

**Comment by Aokromes** [comment]
it's not a submap, it's a different planet.
